lib/wallet/http.js

```diff
@@ -1236,7 +1236,7 @@ class HTTP extends Server {
         return res.json(500, {errors: errors});
 
       if (broadcast) {
-        const tx = await req.wallet.sendMTX(mtx, passphrase);
+        const tx = await req.wallet.sendMTX(mtx, passphrase, true);
```

Comment:
As discussed earlier today, to start, can you use `sendMTXBlocking` specifically for low-value bids? Not for any other transaction type? Let's keep the behavior as similar as we can to how things normally work until we have more confidence in this.

Reply:
fixed ...